scopes/preview/preview/preview.main.runtime.tsx

```diff
@@ -639,9 +642,10 @@ export class PreviewMain {
             return file.path;
           }
           const distRelativePath = compilerInstance.getDistPathBySrcPath(file.relative);
-          return join(this.workspace.path, modulePath, distRelativePath);
+          const moduleName = modulePath.replace(/^.*node_modules\//, '');
```

Comment:
I don't think this is a safe way to get the module name. I think of few cases where this will return wrong module name (like when importing an internal file from a module)

Reply:
Do you have a better approach?
What I received here are some paths like:

```
node_modules/.bit_roots/teambit.envs_env/node_modules/@org/scope-name.envs.my-vue-env
or
node_modules/@teambit/vite.examples.vue.my-component
```

which are in form of `node_modules/<package-name>` or `node_modules/.bit_roots/.../<package-name>`.

Reply:
I reverted these lines of code. 🙏